install/set_gigabyte_node_bmc_to_factory_defaults.md

```diff
@@ -1,111 +1,106 @@
 # Set Gigabyte Node BMC to Factory Defaults
 
-### Prerequisites
+## Prerequisites
+
+**Note**: This section refers to scripts that exist only in the PIT environment. If necessary, copy the LiveCD data from a different machine to get these scripts.
 
 Use the management scripts and text files to reset Gigabyte BMC to factory default settings. Set the BMC to the factory default settings in the following cases:
```

Comment:
I am confused. This preface makes it sound like the list which follows is going to be a list of situations in which the BMC should be reset to its factory default settings.

The first few entries do sound like errors you could encounter, and so they make sense as reasons why you may need to do the factory reset:
```markdown
- There are problems using the `ipmitool` command and Redfish does not respond
- There are problems using the `ipmitool` command and Redfish is running
- When BIOS or BMC flash procedures fails using Redfish
```

But the rest of the list entries sound like a combination of how to do the reset, and prerequisites for doing it:
```markdown
- Run the `do_bmc_factory_default.sh` script
- Run `ipmitool -I lanplus -U admin -P password -H BMC_or_CMC_IP mc reset cold` and flash it again after 5 minutes (300 seconds).
- If booted from the PIT node:
  - the firmware packages are located in the HFP package provided with the Shasta release
  - the required scripts are located in `/var/www/fw/river/sh-svr-scripts`
  - The firmware packages are located in the HFP package provided with the Shasta release
  - The required scripts are located in `/var/www/fw/river/sh-svr-scripts`
```

These latter items belong in a different list.